qsirecon/workflows/base.py

```diff
@@ -85,7 +85,6 @@ def init_single_subject_recon_wf(subject_id):
         recon_workflow_input_fields,
     )
     from ..interfaces.reports import AboutSummary, SubjectSummary
-    from ..interfaces.utils import GetUnique
```

Comment:
You can get rid of the GetUnique interface completely now, right? I don't think it's used anywhere else.